utils/node_local_test.py

```diff
@@ -1404,7 +1403,7 @@ def run_tests(dfuse):
 
     fname = os.path.join(path, 'test_file3')
 
-    rc = subprocess.run(['dd', 'if=/dev/zero', 'bs=16k', 'count=64',
+    rc = subprocess.run(['dd', 'if=/dev/zero', 'bs=16k', 'count=64', # nosec
```

Comment:
(pylint-subprocess-run-check) Using subprocess.run without explicitly set `check` is not recommended.